hack/e2e-pool-test.sh

```diff
@@ -218,6 +218,7 @@ go run "${SRC_ROOT}/contrib/cmd/hiveutil/main.go" clusterpool create-pool \
   ${REGION_ARG} \
 	${CREDS_FILE_ARG} \
 	--pull-secret-file="${PULL_SECRET_FILE}" \
+	--base-domain="${CLUSTER_DOMAIN}" \
```

Comment:
ntr: CLUSTER_DOMAIN is set based on BASE_DOMAIN depending whether we're [using managed DNS](https://github.com/openshift/hive/blob/5bbb1928ca9960e9641794d19035a7d8bdc6aac4/hack/e2e-common.sh#L219) or [not](https://github.com/openshift/hive/blob/5bbb1928ca9960e9641794d19035a7d8bdc6aac4/hack/e2e-common.sh#L234)